nshlib/nsh_console.h

```diff
@@ -153,6 +155,15 @@ struct console_stdio_s
   size_t varsz;
 #endif
 
+#ifdef CONFIG_NSH_ALIAS
```

Comment:
it's better to move into nsh_vtbl_s, so all possible console can get alias support automatically

Reply:
Sure, I did not really know where to put the aliases. I noticed that NSH_VARS are here, so I thought this would be the place.

Can you explain to me the difference between `nsh_vtbl_s` and `console_stdio_s`, it was not totally clear to me?

Reply:
you can think nsh_vtbl_s is a base class and console_stdio_s is a derived class(handle the different I/O), more derived classes may add later. So if this is a common feature for all derived class, nsh_vtbl_s is a better place.

Reply:
Thanks for the explanation.